app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -0,0 +1,163 @@
+package com.wire.android.ui.home.conversations
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.SmallTopAppBar
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBarDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.wire.android.R
+import com.wire.android.ui.common.BackNavigationIconButton
+import com.wire.android.ui.common.OnDropDownIconButton
+import com.wire.android.ui.common.SurfaceBackgroundWrapper
+import com.wire.android.ui.common.button.WireSecondaryButton
+import com.wire.android.ui.home.conversations.mock.mockMessages
+import com.wire.android.ui.home.conversations.model.ConversationView
+import com.wire.android.ui.home.conversations.model.Message
+import com.wire.android.ui.theme.title01
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ConversationScreen(
+    conversationView: ConversationView,
+    onBackButtonClick: () -> Unit
+) {
+    with(conversationView) {
+        Scaffold(
+            topBar = { ConversationScreenTopAppBar(name, onBackButtonClick, {}, {}, {}) },
+            content = {
+                ConversationScreenContent(messages = messages)
+            })
+    }
+}
+
+@Composable
+private fun ConversationScreenContent(messages: List<Message>) {
+    SurfaceBackgroundWrapper {
+        LazyColumn {
+            items(messages) { message ->
+                MessageItem(message = message)
+            }
+        }
+    }
+}
+
+@Composable
+private fun ConversationScreenTopAppBar(
```

Comment:
Maybe extract it to a separate file as this belongs more to navigation than to conversation rendering?